README.md

```diff
@@ -47,7 +52,7 @@ This guide is available in other languages too. See [Translation](#translation)
   1. [ECMAScript 5 Compatibility](#ecmascript-5-compatibility)
   1. [ECMAScript 6+ (ES 2015+) Styles](#ecmascript-6-es-2015-styles)
   1. [Standard Library](#standard-library)
-  1. [Testing](#testing)
+  1. [Code Quality Guardians  <img src="https://emojipedia-us.s3.dualstack.us-west-1.amazonaws.com/thumbs/120/google/241/shield_1f6e1.png" height="13">](#code-quality-guardians)
```

Comment:
it needs more fun, try https://lh3.googleusercontent.com/proxy/_thsLx4wkrRtgVXYw3GkvzqO-mi0zEwkapaxWN4HSaFU2Ky_ZysiEp88QbjSRNrKjj7CPwtgLSPY7_YjfGYCHTc92uKXcXE--VH8kEU9NdQQEhaY8wv6STARdw8OfFdEOgbMnJAIYqUciw